pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1010,6 +1018,28 @@ func getJmsConfig(environment api.EnvironmentType, jms *api.KieAppJmsObject) (*a
 	return jms, nil
 }
 
+func getKafkaConfig(environment api.EnvironmentType, kafka *api.KafkaExtObject) (*api.KafkaExtObject, error) {
+	envConstants := constants.EnvironmentConstants[environment]
```

Comment:
any reason you're pulling the envConstants here? doesn't seem you've added any kafka specific items to that map, so maybe not needed?

Reply:
yep, remove